src/features/serde/mod.rs

```diff
@@ -216,6 +218,24 @@ where
     }
 }
 
+impl<T> std::fmt::Debug for Compat<T>
```

Comment:
I think this can be a simple `#[derive(Debug)]`, but I'm not too bothered by it. It should be `core::fmt::Debug` though